undertow/src/main/java/org/wildfly/extension/undertow/HttpsListenerResourceDefinition.java

```diff
@@ -60,14 +60,14 @@
             .setCapabilityReference(SSL_CONTEXT_CAPABILITY, LISTENER_CAPABILITY_NAME, true)
             .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
             .setValidator(new StringLengthValidator(1))
-            .setAccessConstraints(SensitiveTargetAccessConstraintDefinition.SECURITY_REALM_REF)
```

Comment:
Do we need some sort of SSL_CONTEXT_REF constraint?

This question is OT to this PR even if we do need one. It was just that seeing this bit of the diff sparked the question.